### DISCUSSION TITLE
Is 0.20.0 the last stable version until 1.0?

### DISCUSSION THREAD

**Discussion by alexeiz** [hylang/hy#2113] [Q&A] [answered]
Or can there be 0.21.0, etc?

**Comment by allison-casey** [comment, marked as answer]
0.20 will likely be the last stable release before 1.0. you can check the progress on 1.0 [here](https://github.com/hylang/hy/projects/1). there are a number of breaking changes we're trying to nail down in the alphas, but we've made some pretty good progress.

**Comment by cclauss** [comment]
Given that the URL above now points to an empty project and given that homebrew/homebrew-core#86610 is one of the few PR holding back a full migration to Python 3.10, would it be possible to get a new hy release supporting Python 3.10?
